### PR TITLE
Add RWD Region 2 Option To UI

### PR DESCRIPTION
**Pull request by arottersman**
## Overview

In https://github.com/WikiWatershed/rapid-watershed-delineation/pull/37 we added a new endpoint to the rapid watershed delineation api `rwd-nhd` that allows the delineation of points within NHD Region 2 (the Mid-Atlantic). This PR adds an option in the UI to access it.

screen shot 2016-11-08 at 1 40 25 pm

screen shot 2016-11-08 at 1 41 13 pm

## Notes
In order to check on the front end if we're able to delineate a clicked point with the NHD data, I've added GeoJSON of a simplified, buffered perimeter of Region 2. 
screen shot 2016-11-08 at 1 40 09 pm

I created it in QGIS from the `BoundaryUnit.shp` found [here](http://www.horizon-systems.com/nhdplus/V2GlobalData.php) (by @mmcfarland !) . I performed two vector operations: a "Geoprocessing Tools --> Buffer" with buffer distance 0.10, and a "Geometry Tools --> Simplify Geometries" with simplify tolerance 0.01. 

## Testing
- Pull this branch and get them VM's rolling
- Mount the RWD data if you have it (note the returned region2 shapes might not be accurate, see[ our little projection issue](https://github.com/WikiWatershed/rapid-watershed-delineation/issues/39))
  - If not, you can print the `response_json` [here](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/tasks.py#L50) and run `./scripts/debugcelery.sh`
      - When you select a point confirm the RWD script is at least looking in the right directory: 
`No such file or directory: \'/opt/rwd-data/drb/Main_Watershed\'\n', u'error': u''}`
 or
`No such file or directory: \'/opt/rwd-data/nhd/Main_Watershed\'\n', u'error': u''}`

Now,
- `./scripts/bundle.sh`
- In the draw menu select "Delineate Watershed --> DRB Stream Network" and confirm things work as usual
 - In the draw menu select "Delineate Watershed --> NHD Mid Atlantic Stream Network" and confirm you can click anywhere inside the mid-atlantic area shown below and kick off a job.
screen shot 2016-11-08 at 2 04 25 pm
 - Any click outside the mid-atlantic should show an NHD specific error modal and not kick of a job

Connects #1601

### CODE REVIEW

File: src/mmw/js/src/draw/templates/delineationOptions.html [hunk 1]
```diff
@@ -12,8 +12,14 @@
   </button>
   <ul class="dropdown-menu menu-left" role="menu">
     <li role="presentation">
-      <a role="menuitem" tabindex="-1" href="#" data-shape-type="stream" data-snapping-on="true">
-        <span>Stream Network</span> <i class="split fa fa-question-circle"
+      <a role="menuitem" tabindex="-1" href="#" data-shape-type="stream" data-snapping-on="true" data-data-source={{DRB}}>
+        <span>DRB Stream Network</span> <i class="split fa fa-question-circle"
+          data-content="This automatically snaps to the nearest point on a perennial stream and calculates the watershed boundaries for the stream and its tributaries upstream of the selected point."></i>
```
Comment (mmcfarland) [inline src/mmw/js/src/draw/templates/delineationOptions.html]: We'll need to differentiate the text here between DRB and NHD Region 2, but let's leave it as is and ask the client for input so we don't have to update it twice.

**Comment by arottersman** [comment]
@azavea-bot rebuild

**Comment by arottersman** [comment]
Thanks for taking a look at this, and good point about the tooltip text.

**Comment by arottersman** [comment]
I think the build is failing from a flake8 error - going to force push a fixup to address:
```
/opt/app/apps/modeling/views.py:183:80: E501 line too long (80 > 79 characters)
/opt/app/apps/modeling/views.py:406:80: E501 line too long (84 > 79 characters)
```

**Pull request merged**